### DISCUSSION TITLE
HTTP/2 403  returned ./vendor/bin/rr get-binary

### DISCUSSION THREAD

**Discussion by hemant17** [roadrunner-server/roadrunner#1276] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/10548023/188659439-9a1cbbf2-b1d7-4333-87fd-08b0b9321627.png)

 HTTP/2 403  returned for "https://api.github.com/repos/roadrunner-server/roadrunner/releases?page=1".
while installing with laravel not able to get binary 
any one facing the same issues?

**Comment by rustatian** [comment, marked as answer]
Hey @hemant17 👋🏻 
It might be a GitHub API rate limiter for anonymous calls.

EDIT: You may set a `GITHUB_TOKEN` env variable with a GitHub token (any scope) and then re-run the command: https://github.com/spiral/roadrunner-cli/blob/master/src/Command.php#L39-L45
